Jenkinsfile

```diff
@@ -43,14 +29,37 @@ pipeline {
           def slackURL = env.SLACK_URL
           def sentryDsn = env.SENTRY_DSN
           def gaTrackingId = env.GA_TRACKING_ID
+          def buildNumber = env.BUILD_NUMBER
+          def gitCommit = env.GIT_COMMIT
+          def jobName = env.JOB_NAME
+          def disableDeploy = env.DISABLE_DEPLOY
```

Comment:
Does this need a default value? Wondering if `${14}, ${15}` etc would fail in `hab-build-and-push` if it's an empty string. At some point we might want to consider moving away from positional arguments in our bash scripts (and instead used named arguments).

Reply:
Good catch. It will be a problem when we introduce new variables. I think it would be nice if we could use environment variables instead of expanding the script to support named arguments. We'll have to figure out if that's easy, but it should be doable.

Reply:
Actually, this is fine. It works out correctly because we've quoted the arguments below when calling the script.